connection_configuration_test.go

```diff
@@ -0,0 +1,293 @@
+// Copyright (c) 2024 Snowflake Computing Inc. All rights reserved.
+
+package gosnowflake
+
+import (
+	"io/fs"
+	"os"
+	path "path/filepath"
+	"testing"
+	"time"
+)
+
+func TestTokenFilePermission(t *testing.T) {
+	if !isWindows {
+		os.Setenv(snowflakeHome, "./test_data")
+		_, err := LoadConnectionConfig()
+		assertNotNilF(t, err, "The error should occur because you the permission is not 0600")
```

Comment:
Could we assert the error message?

Reply:
I applied to it. Please take a look. Thank you.